danPortfolio.css

```diff
@@ -0,0 +1,349 @@
+@import url('https://fonts.googleapis.com/css2?family=Itim&display=swap');
+
+@font-face {
+    font-family: "Itim";
+    src: url("../fonts/Itim-Regular.ttf") format('truetype');
+}
+
+* {
+	font-family: 'Itim';
+	box-sizing: border-box;
+	
+}
+
+body {
+	align-self: flex-start;
```

Comment:
this does not do anything because the property is only relevant in a flex context